introducing-java/src/test/java/com/onto/javacourse/intro/arrayReverse/ArrayReverseTest.java

```diff
@@ -0,0 +1,19 @@
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class ArrayReverseTest {
+    @Test
+    public void arrayReverseTest() {
```

Comment:
You need to cover the if-statement in the logic with tests too. What is array is null? What if its length is 0?